### PR TITLE
switch to pulumiverse namespace

### PR DESCRIPTION
**Pull request by jaxxstorm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,29 +13,29 @@ This package is available in many languages in the standard packaging formats.
 To use from JavaScript or TypeScript in Node.js, install using either `npm`:
 
 ```
-$ npm install @jaxxstorm/pulumi-scaleway
+$ npm install @pulumiverse/pulumi-scaleway
```
Comment (ringods) [inline README.md:16]: NPM packages typically drop the `pulumi-` part in the package name. Can you align and update this to `@pulumiverse/scaleway`?

File: provider/cmd/pulumi-resource-scaleway/schema.json [hunk 2]
```diff
@@ -4,15 +4,16 @@
...
     ],
     "homepage": "https://leebriggs.co.uk/projects#pulumi-scaleway",
     "license": "Apache-2.0",
     "attribution": "This Pulumi package is based on the [`scaleway` Terraform Provider](https://github.com/scaleway/terraform-provider-scaleway).",
-    "repository": "https://github.com/jaxxstorm/pulumi-scaleway",
+    "repository": "https://github.com/pulumiverse/pulumi-scaleway",
     "logoUrl": "https://raw.githubusercontent.com/jaxxstorm/pulumi-scaleway/master/assets/scaleway-svgrepo-com.svg",
-    "pluginDownloadURL": "https://github.com/jaxxstorm/pulumi-scaleway/releases/download/${VERSION}",
-    "publisher": "Lee Briggs",
+    "pluginDownloadURL": "https://github.com/pulumiverse/pulumi-scaleway/releases/download/${VERSION}",
```
Comment (ringods) [inline provider/cmd/pulumi-resource-scaleway/schema.json:15]: I know Github Releases support requires Pulumi 3.35.3 or up, however, I would like to remain consistent with the other providers in Pulumiverse. Could you update the URL to `github://api.github.com/pulumiverse`?

**Review by ringods — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jaxxstorm) [reply README.md]: Fixed in latest commit

File: provider/cmd/pulumi-resource-scaleway/schema.json (hunk 2 above)
Comment (jaxxstorm) [reply provider/cmd/pulumi-resource-scaleway/schema.json]: Fixed

File: docs/_index.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ The Scaleway provider must be configured with credentials to deploy and update r
 {{% choosable language typescript %}}
 
 ```typescript
-import * as scaleway from "@jaxxstorm/pulumi-scaleway";
+import * as scaleway from "@pulumiverse/pulumi-scaleway";
```
Comment (ringods) [inline docs/_index.md:16]: @jaxxstorm this example still needs an update to `@pulumiverse/scaleway`

File: provider/resources.go [hunk 4]
```diff
@@ -216,7 +214,7 @@ func Provider() tfbridge.ProviderInfo {
 				"@types/node": "^10.0.0", // so we can access strongly typed node definitions.
 				"@types/mime": "^2.0.0",
 			},
-			PackageName: "@jaxxstorm/pulumi-scaleway",
+			PackageName: "@pulumiverse/pulumi-scaleway",
```
Comment (ringods) [inline provider/resources.go:217]: @jaxxstorm package name should be `@pulumiverse/scaleway`

**Review by ringods — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ringods) [reply README.md]: @jaxxstorm I still see the wrong package name popping up.

File: docs/_index.md (hunk 3 above)
Comment (ringods) [inline docs/_index.md:16]: Package name must have the `pulumi-` part dropped.

File: docs/installation-configuration.md [hunk 5]
```diff
@@ -8,9 +8,9 @@ layout: installation
 
 The Pulumi Scaleway provider is available as a package in all Pulumi languages:
 
-* JavaScript/TypeScript: [`@jaxxstorm/pulumi-scaleway`](https://www.npmjs.com/package/@jaxxstorm/scaleway)
+* JavaScript/TypeScript: [`@pulumiverse/pulumi-scaleway`](https://www.npmjs.com/package/@pulumiverse/scaleway)
```
Comment (ringods) [inline docs/installation-configuration.md:11]: Package name must have the `pulumi-` part dropped.

File: examples/examples_nodejs_test.go [hunk 6]
```diff
@@ -25,7 +25,7 @@ func getJSBaseOptions(t *testing.T) integration.ProgramTestOptions {
 	baseJS := base.With(integration.ProgramTestOptions{
 		ExpectRefreshChanges: true,
 		Dependencies: []string{
-			"@jaxxstorm/pulumi-scaleway",
+			"@pulumiverse/pulumi-scaleway",
```
Comment (ringods) [inline examples/examples_nodejs_test.go:28]: Package name must have the `pulumi-` part dropped.

File: examples/ts/server/index.ts [hunk 7]
```diff
@@ -1,5 +1,5 @@
 import * as pulumi from "@pulumi/pulumi";
-import * as scaleway from "@jaxxstorm/pulumi-scaleway";
+import * as scaleway from "@pulumiverse/pulumi-scaleway";
```
Comment (ringods) [inline examples/ts/server/index.ts:2]: Package name must have the `pulumi-` part dropped.

File: examples/ts/server/package.json [hunk 8]
```diff
@@ -4,7 +4,7 @@
         "@types/node": "^14"
     },
     "dependencies": {
-        "@jaxxstorm/pulumi-scaleway": "^0.1.5",
+        "@pulumiverse/pulumi-scaleway": "^0.1.5",
```
Comment (ringods) [inline examples/ts/server/package.json:7]: Package name must have the `pulumi-` part dropped.

File: provider/cmd/pulumi-resource-scaleway/schema.json [hunk 9]
```diff
@@ -35,7 +36,7 @@
             },
             "disableUnionOutputTypes": true,
             "packageDescription": "A Pulumi package for creating and managing scaleway cloud resources.",
-            "packageName": "@jaxxstorm/pulumi-scaleway",
+            "packageName": "@pulumiverse/pulumi-scaleway",
```
Comment (ringods) [inline provider/cmd/pulumi-resource-scaleway/schema.json:39]: Package name must have the `pulumi-` part dropped.

File: provider/resources.go (hunk 4 above)
Comment (ringods) [reply provider/resources.go:217]: Package name must have the `pulumi-` part dropped to be similar to all the other generated Pulumi JS/TS packages.

**Review by ringods — CHANGES_REQUESTED** [review]

**Review by ringods — APPROVED** [review]

**Pull request merged**